src/owaspUtils.ts

```diff
@@ -741,27 +754,30 @@ export function toBNbasic( x?: any, optionalRadix?: number ): any {
 export function toBN( arg: any ): any {
     if( typeof arg === "string" || typeof arg === "number" ) {
         let multiplier = toBNbasic( 1 );
-        const formattedString = String( arg ).toLowerCase().trim()
+        const formattedString = String( arg ).toLowerCase().trim();
         const isHexPrefixed =
             formattedString.substr( 0, 2 ) === "0x" ||
-            formattedString.substr( 0, 3 ) === "-0x"
+            formattedString.substr( 0, 3 ) === "-0x";
         let stringArg = stripHexPrefix( formattedString );
         if( stringArg.substr( 0, 1 ) === "-" ) {
-            stringArg = stripHexPrefix( stringArg.slice( 1 ) )
+            stringArg = stripHexPrefix( stringArg.slice( 1 ) );
             multiplier = toBNbasic( -1, 10 );
         }
         stringArg = stringArg === "" ? "0" : stringArg;
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why have you disabled this rule?

Reply:
Fixed

Reply:
Yes. I use it in some cases to avoid modification reg-exps

Reply:
It does not affect regexps, so better to remove it

Reply:
Fixed